lntop: update to v0.2.0

Stadicus opened the pull request:
#### What

Update to latest release now that binaries are available.

### Why

Version 0.2.0 of lntop adds some new features and bugfixes:  
https://github.com/edouardparis/lntop/releases/tag/v0.2.0

This change also adds binary signature verification as best practice.

### How

Simple update of download links, plus added instructions for checksum signature validation. See https://github.com/edouardparis/lntop/issues/49 for some context.

Deleting downloaded files is removed as these are small and the /tmp folder is purged on every reboot anyway.

### Scope

- [ ] significant change to core configuration
- [ ] independent bonus guide
- [x] simple application update

### Test & maintenance

Follow instructions and run `lntop`

VajraOfIndra commented on bonus/lightning/lntop.md:
````diff
@@ -7,42 +7,86 @@ nav_exclude: true
+* Verify the signed checksum against the actual checksum of your download
+
+  ```sh
+  $ $ sha256sum --check checksums-lntop-v0.2.0.txt --ignore-missing
````
```suggestion
  $ sha256sum --check checksums-lntop-v0.2.0.txt --ignore-missing
```
Fixed typo

Stadicus replied:
Ack, thanks!

VajraOfIndra commented on bonus/lightning/lntop.md:
````diff
@@ -7,42 +7,86 @@ nav_exclude: true
 $ lntop
 ```
+
 ------
````
Nitpicking here, but is there any reason why there is 6 - there and not 3 as usual?

Stadicus replied:
No difference, 3x `-` have the same effect.

VajraOfIndra commented on bonus/lightning/lntop.md:
````diff
@@ -7,42 +7,86 @@ nav_exclude: true
+
+---
 
 ### Run lntop
 
 ```bash
 $ lntop
 ```
````
```suggestion
### Run lntop

```bash
$ lntop
```

* Note: If you have a large LND database, it might take a while (up to 10-20 seconds) for the dashboard to appear. You'll see a blank screen while it's loading, do not worry!
```

I thought I had an issue with the install because I got a black screen when I ran lntop. The first couple of times I Ctrl+C before it had time to load! I assume it's dependent on` channel.db` size... So I suggest we add a little note to warn users it might  take a while for the dashboard to load. It takes about 10 seconds for me right now.

But if you think that's overkill, no problem, we can just let the user figure it out.

Stadicus replied:
Good point

VajraOfIndra reviewed (changes requested):
Nice update! one more bonus guide which is up-to-date for v3 :1st_place_medal: 

A couple of typo fixes and one suggestion to reassure user that the dashbaord can take a while to load.

Other than that, I think it could be beneficial to add a section that explains how to navigate the dashboard with the keyboard, using F1 (help), F2 (left menu), Enter, q (quit), a (sort column, ascending), d (sort column, descending). I think that'd be super helpful, particulalry because it's not easy to find the explanations. But we could do this with another PR later.

Other than that, I'm puzzled regarding the Go requirement. I thought Go was _not_installed by default on RasPiOS Lite.. so how come we can use lntop without installing Go first?

Stadicus commented:
Addressed all your feedback, thanks!

> Other than that, I think it could be beneficial to add a section that explains how to navigate the dashboard with the keyboard, using F1 (help), F2 (left menu), Enter, q (quit), a (sort column, ascending), d (sort column, descending).

Good idea! I think that could be another pull-request, though. We could open an issue for this as a reminder, with the "help wanted" tag.

PTAL

VajraOfIndra reviewed (approved):
LGTM!
